modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/indicator/test/ChangeTrackingIndicatorConflictIconTest.java

```diff
@@ -0,0 +1,258 @@
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import java.io.ByteArrayOutputStream;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Noor Najjar
+ */
+@RunWith(Arquillian.class)
+public class ChangeTrackingIndicatorConflictIconTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setProductionModeWithSafeCloseable()) {
+
+			_journalArticle = JournalTestUtil.addArticle(
+				_group.getGroupId(), RandomTestUtil.randomString(),
+				RandomTestUtil.randomString());
+		}
+
+		_ctCollection1 = _ctCollectionLocalService.addCTCollection(
+			null, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			0, "P1", null);
+		_ctCollection2 = _ctCollectionLocalService.addCTCollection(
+			null, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			0, "P2", null);
+	}
+
+	@Test
+	public void testConflictIconWithConflict() throws Exception {
+		JournalArticle modifiedArticle = null;
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					_ctCollection1.getCtCollectionId())) {
+
+			modifiedArticle = JournalTestUtil.updateArticle(
+				_journalArticle, "testModifyJournalArticle");
+		}
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setProductionModeWithSafeCloseable()) {
+
+			JournalTestUtil.updateArticle(
+				_journalArticle, "testModifyJournalArticle");
+		}
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					_ctCollection1.getCtCollectionId())) {
+
+			MockLiferayResourceResponse mockLiferayResourceResponse =
+				new MockLiferayResourceResponse();
+
+			_mvcResourceCommand.serveResource(
+				_getMockLiferayResourceRequest(
+					_portal.getClassNameId(JournalArticle.class),
+					modifiedArticle.getPrimaryKey(),
+					_ctCollection1.getCtCollectionId()),
+				mockLiferayResourceResponse);
+
+			JSONObject jsonObject = _getConflictInfoJSONObject(
+				mockLiferayResourceResponse);
+
+			Assert.assertEquals(
+				"change-tracking-conflict-icon-danger",
+				String.valueOf(jsonObject.get("conflictIconClass")));
+			Assert.assertEquals(
+				"warning-full",
+				String.valueOf(jsonObject.get("conflictIconName")));
+		}
```

Comment:
I see these same steps of assertions in every test cases. Let's extract it to a helper method (i.e. `_assertGetConflictInfo(String expectedConflictIconClass, String expectedConflictIconName)` to reduce duplicate code.